tox.ini

```diff
@@ -19,12 +18,16 @@ setenv =
 deps =
     -rrequirements.txt
     -rtest-requirements.txt
-    ansible24: ansible==2.4.6.0
-    ansible25: ansible==2.5.9
-    ansible26: ansible==2.6.4
+    ansible24: ansible>=2.4,<2.5
+    ansible25: ansible>=2.5,<2.6
+    ansible26: ansible>=2.6,<2.7
+    ansible27: ansible>=2.7,<2.8
+    ansibledevel: git+https://github.com/ansible/ansible.git
+extras =
+    azure
 commands =
-    unit: py.test -vv --cov-report=term-missing --cov={toxinidir}/molecule/ --no-cov-on-fail {posargs}
-    functional: py.test -vv -x test/functional/ {posargs}
+    unit: pytest -vv --cov-report=term-missing --cov={toxinidir}/molecule/ --no-cov-on-fail {posargs}
+    functional: pytest -vv -x test/functional/ {posargs}
```

Comment:
@gundalow `-x` means "stop after first failure". Do we need this?

https://docs.pytest.org/en/latest/usage.html#stopping-after-the-first-or-n-failures